### DISCUSSION TITLE
模式和环境变量 里面的 TARO_APP_ID，和taro的多平台机制是不是冲突了？

### DISCUSSION THREAD

**Discussion by p2227** [NervJS/taro#14727] [Q & A]
https://taro-docs.jd.com/docs/next/env-mode-config

特殊环境变量 TARO_APP_ID
信息
Taro v3.6.9 开始支持

TARO_APP_ID 是专门针对小程序的 appid 设计的，在构建输出 dist/project.config.json 文件前， 会将 dist/project.config.json 文件中的 appid 字段，修改为 TARO_APP_ID 的值。 在不同环境配置不同的小程序 appid 时，它特别有用，还能免去开发者在开发者工具上手动修改 appid 的麻烦。

那加了这个配置后，我去编译支付宝小程序、抖音小程序的时候，要怎么配置 这个 TARO_APP_ID 变量？

是不是要配置

.env.dev-tt
.env.dev-alipay

等多个文件？有最佳实践推荐吗？

**Comment by shitoujianzibu** [comment]
同问
